blocks.md

```diff
@@ -384,40 +396,47 @@ Creates a block which displays disk space information.
 [[block]]
 block = "disk_space"
 path = "/"
-alias = "/"
 info_type = "used"
 unit = "GiB"
-format = "{icon}{used}/{total} {unit} ({available}{unit} free)"
+format = "{icon} {used}/{total} ({available} free)"
 ```
 
 #### Options
 
 Key | Values | Required | Default
 ----|--------|----------|--------
 `alert` | Available disk space critical level as a percentage or Unit. | No | `10.0`
-`alias` | Alias that is displayed for path. | No | `"/"`
-`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{alias} {available} {unit}"`
+`warning` | Available disk space warning level as a percentage or Unit. | No | `20.0`
+`format` | A string to customise the output of this block. See below for available placeholders. Text may need to be escaped, refer to [Escaping Text](#escaping-text). | No | `"{alias} {available}"`
 `info_type` | Currently supported options are `"available"`, `"free"`, and `"used"` (sets value for alert and percentage calculation). | No | `"available"`
 `interval` | Update interval, in seconds. | No | `20`
 `path` | Path to collect information from. | No | `"/"`
-`unit` | Unit that is used to display disk space. Options are `"MB"`, `"MiB"`, `"GB"`, `"GiB"`, `"TB"`, `"TiB"` and `"Percent"`. | No | `"GB"`
-`warning` | Available disk space warning level as a percentage or Unit. | No | `20.0`
+`unit` | Unit that is used when `alert_absolute` is set for `warning` and `alert`. Options are `"B"`, `"KB"` `"MB"`, `"GB"`, `"TB"`. | No | `"GB"`
 `alert_absolute` | Use Unit values for warning and alert instead of percentages. | No | `false`
 
+#### Deprecated Options
+Key | Values | Required | Default
+----|--------|----------|--------
+`alias` | Sets the value for `{alias}` placeholder | No | `"/"`
+
+
 #### Available Format Keys
 
-Key | Value
-----|-------
-`{alias}` | Alias for disk path
-`{available}` | Available disk space (free disk space minus reserved system space)
-`{bar}` | Display bar representing percentage
-`{free}` | Free disk space
-`{icon}` | Disk drive icon
-`{path}` | Path used for capacity check
-`{percentage}` | Percentage of disk used or free (depends on info_type setting)
-`{total}` | Total disk space
-`{unit}` | Unit used for disk space (see above)
```

Comment:
Is it possible to keep {unit} around and deprecate it instead? Then we can remove it in a future release. I'm guessing not due to the formatter changes, in that case we need to mention it somewhere.

Reply:
So you want it to be an alias to `unit` config option?

Reply:
Actually, we can just remove it since leaving it around would make the output weird and users would have to look up what changed anyway.

Add it to the list of breaking changes in NEWS.md and we can resolve this that way.